src/sentry/lang/native/plugin.py

```diff
@@ -524,21 +189,17 @@ def process_frame(self, processable_frame, processing_task):
         if processable_frame.cache_value is None:
             # Construct a raw frame that is used by the symbolizer
             # backend.  We only assemble the bare minimum we need here.
-            sym_input_frame = {
-                'object_name': frame.get('package'),
-                'instruction_addr': processable_frame.data['instruction_addr'],
-                'symbol_name': frame.get('function'),
-            }
-            in_app = self.sym.is_in_app(sym_input_frame)
-            raw_frame['in_app'] = in_app
+            instruction_addr = processable_frame.data['instruction_addr']
```

Comment:
We now use the same format in symsynd so this simplified the logic tremendously.